requirements.txt

```diff
@@ -1,3 +1,3 @@
-cirq==0.13.0.dev20210901195302
-qiskit==0.28.0
-scipy==1.6.1
+cirq!=0.13.0.dev20210901195302
```

Comment:
Shouldn't this be:
```suggestion
cirq~=0.13.0.dev20210901195302
```

Reply:
Ended up stealing the requirements from [cirq-superstaq](https://github.com/SupertechLabs/cirq-superstaq/pull/227)